assets/src/js/frontend/give-stripe-becs.js

```diff
@@ -0,0 +1,290 @@
+
+		// Prepare Card Elements for each form on a single page.
+		globalIbanElements[ idPrefix ] = [];
+
+		Array.prototype.forEach.call( bankAccountElementSelectors, function( selector, index ) {
+			globalIbanElements[ idPrefix ][ index ] = [];
+			globalIbanElements[ idPrefix ][ index ].item = bankAccountElements[ index ];
+			globalIbanElements[ idPrefix ][ index ].selector = selector;
+			globalIbanElements[ idPrefix ][ index ].isCardMounted = false;
+		} );
+
+		// Mount and Un-Mount Stripe CC Fields on gateway load.
+		jQuery( document ).on( 'give_gateway_loaded', function( event, xhr, settings ) {
+			// Un-mount card elements when stripe is not the selected gateway.
+			giveStripeUnmountIbanElements( globalIbanElements[ idPrefix ] );
+
+			if ( formElement.querySelector( '.give-gateway-option-selected .give-gateway' ).value === 'stripe_becs' ) {
+				// Mount card elements when stripe is the selected gateway.
+				giveStripeMountIbanElements( idPrefix, globalIbanElements[ idPrefix ] );
+			}
+		} );
+
+		// Mount Card Elements, if default gateway is Stripe BECS.
+		if ( 'stripe_becs' === defaultGateway || give_stripe_vars.stripe_card_update ) {
+			// Disabled the donate button of the form.
+			donateButton.setAttribute( 'disabled', 'disabled' );
+
+			giveStripeMountIbanElements( idPrefix, globalIbanElements[ idPrefix ] );
+
+			// Enable the donate button of the form after successful mounting of CC fields.
+			donateButton.removeAttribute( 'disabled' );
+		} else {
+			giveStripeUnmountIbanElements( bankAccountElements );
+		}
+	} );
+
+	// Process Donation using Stripe Elements on form submission.
+	jQuery( 'body' ).on( 'submit', '.give-form', function( event ) {
+		const $form = jQuery( this );
+		const $idPrefix = $form.find( 'input[name="give-form-id-prefix"]' ).val();
+
+		if ( 'stripe_becs' === $form.find( 'input.give-gateway:checked' ).val() || give_stripe_vars.stripe_card_update  ) {
+			give_stripe_process_becs_bank_account( $form, globalIbanElements[ $idPrefix ][ 0 ].item );
+			event.preventDefault();
+		}
+	} );
+
+	/**
+	 * Mount Card Elements
+	 *
+	 * @param {string} idPrefix     ID Prefix.
+	 * @param {array}  bankAccountElements List of card elements to be mounted.
+	 *
+	 * @since 1.6
+	 */
+	function giveStripeMountIbanElements( idPrefix, bankAccountElements = [] ) {
+		const bankAccountElementsLength = Object.keys( bankAccountElements ).length;
+
+		// Assign any card element to variable to create source.
+		if ( bankAccountElementsLength > 0 ) {
+			card = bankAccountElements[ 0 ].item;
+		}
+
+		// Mount required card elements.
+		Array.prototype.forEach.call( bankAccountElements, function( value, index ) {
+			if ( false === value.isCardMounted ) {
+				value.item.mount( value.selector + idPrefix );
+				value.isCardMounted = true;
+			}
+		} );
+	}
+
+	/**
+	 * Un-mount Card Elements
+	 *
+	 * @param {array} bankAccountElements List of card elements to be unmounted.
+	 *
+	 * @since 1.6
```

Comment:
This file is newly added, are we using correct @since in this file

Reply:
Resolved here: https://github.com/impress-org/givewp/commit/91ce4cad850ef886d5cf00da5420cd5b2709368d